test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -0,0 +1,619 @@
+    private final static int standbyReplicationServerPort = 9020;
+    private final static String activeCorfuEndpoint = DEFAULT_HOST + ":" + activeClusterCorfuPort;
+    private final static String standbyCorfuEndpoint = DEFAULT_HOST + ":" + standbyClusterCorfuPort;
+
+    private Process activeCorfuServer = null;
+    private Process standbyCorfuServer = null;
+    private Process activeReplicationServer = null;
+    private Process standbyReplicationServer = null;
+
+    private CorfuRuntime activeRuntime;
+    private CorfuRuntime standbyRuntime;
+    private CorfuTable<String, Integer> mapActive;
+    private CorfuTable<String, Integer> mapStandby;
+
+    private CorfuStore corfuStore;
+    private Table<CommonTypes.Uuid, CommonTypes.Uuid, CommonTypes.Uuid> configTable;
+
+    @Before
+    public void setUp() throws Exception {
+        activeCorfuServer = runServer(activeClusterCorfuPort, true);
+        standbyCorfuServer = runServer(standbyClusterCorfuPort, true);
+
+        CorfuRuntime.CorfuRuntimeParameters params = CorfuRuntime.CorfuRuntimeParameters
+                .builder()
+                .build();
+
+        activeRuntime = CorfuRuntime.fromParameters(params).setTransactionLogging(true);
+        activeRuntime.parseConfigurationString(activeCorfuEndpoint).connect();
+
+        standbyRuntime = CorfuRuntime.fromParameters(params).setTransactionLogging(true);
+        standbyRuntime.parseConfigurationString(standbyCorfuEndpoint).connect();
+
+        mapActive = activeRuntime.getObjectsView()
+                .build()
+                .setStreamName(streamName)
+                .setTypeToken(new TypeToken<CorfuTable<String, Integer>>() {
+                })
+                .open();
+
+        mapStandby = standbyRuntime.getObjectsView()
+                .build()
+                .setStreamName(streamName)
+                .setTypeToken(new TypeToken<CorfuTable<String, Integer>>() {
+                })
+                .open();
+
+        assertThat(mapActive.size()).isZero();
+        assertThat(mapStandby.size()).isZero();
+
+        corfuStore = new CorfuStore(activeRuntime);
+
+        configTable = corfuStore.openTable(
+                DefaultClusterManager.CONFIG_NAMESPACE, DefaultClusterManager.CONFIG_TABLE_NAME,
+                CommonTypes.Uuid.class, CommonTypes.Uuid.class, CommonTypes.Uuid.class,
+                TableOptions.builder().build()
+        );
+    }
+
+    @After
+    public void tearDown() throws IOException, InterruptedException {
+        if (activeRuntime != null) {
+            activeRuntime.shutdown();
+        }
+
+        if (standbyRuntime != null) {
+            standbyRuntime.shutdown();
+        }
+
+        shutdownCorfuServer(activeCorfuServer);
+        shutdownCorfuServer(standbyCorfuServer);
+        shutdownCorfuServer(activeReplicationServer);
+        shutdownCorfuServer(standbyReplicationServer);
+    }
+
+    /**
+     * This test verify config change with a role switch.
+     * <p>
```

Comment:
These tests are good! I think one that we might be missing and could be interesting is:
- Snapshot and Log Entry (check it completes)
- Stop both LRs
- Enforce Role Switch
- Bring them back

This way we can verify it performs correctly the site flip even when coming back up after with a new config.